README.md

```diff
@@ -215,11 +258,13 @@ Please make sure to include `#gene_id` in the first line. The default exclusion
 
 There are 5 different output files produced by `staramr`:
 
-1. `summary.tsv`:  A summary of all detected AMR genes/mutations in each genome, one genome per line.
-2. `resfinder.tsv`: A tabular file of each AMR gene and additional BLAST information from the **ResFinder** database, one gene per line.
-3. `pointfinder.tsv`: A tabular file of each AMR point mutation and additional BLAST information from the **PointFinder** database, one gene per line.
-4. `settings.txt`: The command-line, database versions, and other settings used to run `staramr`.
-5. `results.xlsx`: An Excel spreadsheet containing the previous 4 files as separate worksheets.
+1. `summary.tsv`:  A summary of all detected AMR genes/mutations/plasmids in each genome, one genome per line.
+2. `detailed_summary.tsv`: A detailed summary of all detected AMR genes/mutations/plasmids in each genome, one gene per line.
+3. `resfinder.tsv`: A tabular file of each AMR gene and additional BLAST information from the **ResFinder** database, one gene per line.
+4. `pointfinder.tsv`: A tabular file of each AMR point mutation and additional BLAST information from the **PointFinder** database, one gene per line.
+5. `plasmidfinder.tsv`: A tabular file of each AMR plasmid gene and additional BLAST information from the **PlasmidFinder** database, one plasmid gene per line.
+6. `settings.txt`: The command-line, database versions, and other settings used to run `staramr`.
+7. `results.xlsx`: An Excel spreadsheet containing the previous 4 files as separate worksheets.
```

Comment:
previous **4** files -> previous **6** files

Reply:
Solved in commit 2202e45